streams-activitypub-webapp/src/test/scala/org/apache/streams/activitypub/webapp/test/ActivityPubWebappTestSuite.scala

```diff
@@ -13,15 +13,18 @@ import org.junit.platform.suite.api.SuiteDisplayName;
 object ActivityPubWebappTestSuite {
 
   @RegisterExtension
-  val helper: ActivityPubWebappTestSuiteExtension = new ActivityPubWebappTestSuiteExtension()
+  given webapp: ActivityPubWebappTestSuiteExtension = new ActivityPubWebappTestSuiteExtension()
+
+  @RegisterExtension
+  given graph: ActivityPubGraphTestSuiteExtension = new ActivityPubGraphTestSuiteExtension()
 
```

Comment:
ActivityPubGraphTestSuiteExtension is part of the test-jar, so it can be trivially launched to support webapp module test suite as well as graph module test suite 💪